src/shared/constants.ts

```diff
@@ -3,3 +3,7 @@ export const cacheKeys = {
   accountBalance: 'accountBalance',
   tokenAllowance: 'tokenAllowance',
 };
+
+export const FIFTEEN_MINUTES_IN_MILLISECONDS = 15 * 60 * 1000;
```

Comment:
Maybe it makes more sense to have a constant `MINUTES_IN_MILLISECONDS` and then multiply by `15` in the places where 15 minutes are needed.